### ISSUE TITLE
Can't find cfg for librespeech?

### ISSUE THREAD

**Issue by Nimitz14** [mravanelli/pytorch-kaldi#14]
Hey,
trying to reproduce the results for the twinreg paper. The cfg folder only seems to contain stuff for TIMIT though?

Also in the paper it says you just ran the kaldi recipe for the corpus, does that mean you trained alignments using the full ~900 hour dataset (as is done in the current kaldi recipe for librespeech)? In the paper it says you only used the 100 hour subset (train-clean), was that only for training the DNN part (and I'm guessing you did not use speed perturbation, so it really is 100 hours in the end?)?

**Comment by mravanelli** [comment]
Dear Rufolf,
thank you for the interest into the twinreg paper. Yes, in the current
repository we only added the configuration files for the TIMIT experiments.
Note that in the next week we will release a new version of the
pytorch-kaldi project where, beyond many improvements, we will also add
librispeech baselines. For librispeech experiments, we used the training
subset composed of 100 hours. From the 100-h dataset we trained the kaldi
DNN and we use it to derive alignments for the test-clean and dev-clean
datasets. We didn't use the full 900 hours dataset or speech perturbation
techniques.

Mirco
